src/reader/callback/events.cpp

```diff
@@ -142,48 +141,35 @@ namespace reader
                     return static_cast<OTF2_CallbackCode>(OTF2_SUCCESS);
                 }
 
-                typedef otf2::event::metric::value_container value_container;
+                otf2::chrono::time_point timestamp =
+                    otf2::chrono::convert(reader->ticks_per_second())(otf2::chrono::ticks(
```

Comment:
In hindsight, this conversion code is disgusting. We need to make a separate PR and move all occurrences into a function.